src/libplctag/DataTypes/StringMarshaller.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Text;
+
+namespace libplctag.DataTypes
+{
+    public class StringMarshaller : IMarshaller<string>
+    {
+
+        const int MAX_LOGIX_STRING_LENGTH = 82;
+        const int MAX_PCCC_STRING_LENGTH = 80;
+
+        public PlcType PlcType { get; set; }
+
+        public int? ElementSize
```

Comment:
@kyle-github - this is one that I'm not sure about.

This works for my CompactLogix, but I don't know how to make this work for other Plc types.